.env.test

```diff
@@ -0,0 +1,30 @@
+# This file is a "template" of which env vars need to be defined for your application
```

Comment:
What do you think about keeping in `.env.test` only the diff drom `.env`? So, if MAILER_URL is different, then we add it to this file. In case of `APP_SECRET`, I think it can be removed from this file and so on.